packages/marketplace/contracts/interfaces/ITransferManager.sol

```diff
@@ -13,6 +13,7 @@ abstract contract ITransferManager {
     struct DealSide {
         LibAsset.Asset asset; // The asset associated with this side of the deal.
         address account; // The account address associated with this side of the deal.
+        address makeRecipient; // The account address receiving the tokens
```

Comment:
no need to keep the notion of maker / taker in transfer manager
makeRecipient => recipient